tracecat/actions/core/http.py

```diff
@@ -20,6 +20,15 @@ class HTTPResponse(TypedDict):
     data: str | dict[str, Any] | list[Any] | None
 
 
+def get_jwt_token(
+    url: str,
+    json: dict[str, str] | None = None,
+    headers: dict[str, str] | None = None,
+) -> str:
+    token = httpx.get(url, json=json, headers=headers).json()["token"]
+    return token
```

Comment:
This should be async, otherwise you block the entire worker thread across concurrent workflows/actions in the GET

Reply:
Perhaps add some error handling on this as well